.gitignore

```diff
@@ -0,0 +1,9 @@
+# editor and IDE paraphernalia
+.idea
+*.swp
+*.swo
+*~
+.vscode
+
+# MacOS related artifacts
+.DS_Store
```

Comment:
nit: newline.

also, 

*.test
*.out
*.log

Reply:
I'm not sure those apply, since there is only `yaml` `sh` `png` in this repo. I'll differ to the maintainers.